drivers/wdm/audio/backpln/portcls/pin_wavert.cpp

```diff
@@ -311,18 +311,28 @@ CPortPinWaveRT::DeviceIoControl(
     }
     else if (IoStack->Parameters.DeviceIoControl.IoControlCode == IOCTL_KS_ENABLE_EVENT)
     {
-        /// FIXME
-        /// handle enable event
+        /* FIXME UNIMPLEMENTED */
+        UNIMPLEMENTED_ONCE;
     }
     else if (IoStack->Parameters.DeviceIoControl.IoControlCode == IOCTL_KS_DISABLE_EVENT)
     {
-        /// FIXME
-        /// handle disable event
+        /* FIXME UNIMPLEMENTED */
+        UNIMPLEMENTED_ONCE;
     }
+    else if (IoStack->Parameters.DeviceIoControl.IoControlCode == IOCTL_KS_HANDSHAKE)
+	{
+        /* FIXME UNIMPLEMENTED */
+        UNIMPLEMENTED_ONCE;
+	}
+	else if (IoStack->Parameters.DeviceIoControl.IoControlCode == IOCTL_KS_METHOD)
+	{
+        /* FIXME UNIMPLEMENTED */
+        UNIMPLEMENTED_ONCE;
+	}
```

Comment:
Whole block is still misaligned.

Moreover, it looks like all unimplemented cases might be merged, for the time being. But I guess you may want separate line numbers...

Reply:
I'd say separate line numbers so you can actually see what isn't implemented